src/lib/plugins/blocks.js

```diff
@@ -60,14 +62,14 @@ module.exports.player = function (player, serv) {
       if (!results) return false
       return results
     },
-    action (params) {
+    action (params, ctx) {
       let res = params.slice(1, 4)
-      res = res.map((val, i) => serv.posFromString(val, player.position[['x', 'y', 'z'][i]]))
-      player.setBlock(new Vec3(res[0], res[1], res[2]).floored(), params[4], params[5] || 0)
+      res = res.map((val, i) => serv.posFromString(val, ctx.player.position[['x', 'y', 'z'][i]]))
```

Comment:
You need to either do something appropriate if ctx.player is undefined or mark this as playerOnly

Same for all the commands

Reply:
will work on it soon